db/sql/sql.go

```diff
@@ -1213,7 +1213,13 @@ func AddFilterToQuery(s *schema.Schema, q sq.SelectBuilder, filter map[string]in
 				v[i], _ = strconv.ParseBool(j)
 			}
 			q = q.Where(sq.Eq{column: v})
-		} else {
+		} else if !s.SearchLikeDisabled() && len(queryValues) == 1{
+			var searchValue strings.Builder
+			searchValue.WriteString("%")
+			searchValue.WriteString(value.([]string)[0])
```

Comment:
this will panic if value is an array of non-string type (eg. []interface{} or []int).
please add tests.

Reply:
actually I was wrong here, in this branch we (implicitly) know that `value` is indeed a `[]string`. nevertheless, this should go to a separate function.